tests/acceptance/features/apiGraph/getUser.feature

```diff
@@ -1005,3 +1005,223 @@ Feature: get users
+        "driveType": {
+          "type": "string",
+          "enum": ["personal"]
+        },
+        "id": {
+          "type": "string",
+          "pattern": "^%space_id_pattern%$"
+        },
+        "name": {
+          "type": "string",
+          "enum": ["Brian Murphy"]
+        },
+        "webUrl": {
+          "type": "string",
+          "pattern": "^%base_url%/f/%space_id_pattern%$"
+        },
+        "owner": {
+          "type": "object",
+          "required": [
+            "user"
+          ],
+          "properties": {
+            "user": {
+              "type": "object",
+              "required": [
+                "displayName",
+                "id"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "enum": [""]
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                }
+              }
+            }
+          }
+        },
+        "qouta": {
+          "type": "object",
+          "required": [
+            "state"
+          ],
+          "properties": {
+            "state": {
+              "type": "string",
+              "enum": ["normal"]
+            }
+          }
+        },
+        "root": {
+          "type": "object",
+          "required": [
+            "webDavUrl"
+          ],
+          "properties": {
+            "webDavUrl": {
+              "type": "string",
+              "pattern": "^%base_url%/dav/spaces/%space_id_pattern%$"
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | userRole    |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | Guest       |
+
+
+  Scenario Outline: non-admin user tries to get drive information of other user with different user role
+    Given the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    And the administrator has assigned the role "<userRole>" to user "Brian" using the Graph API
+    When user "Alice" gets the personal drive information of user "Brian" using Graph API
+    Then the HTTP status code should be "404"
+    Examples:
+      | user-role   | userRole    |
+      | Space Admin | Admin       |
+      | Space Admin | Space Admin |
+      | Space Admin | User        |
+      | Space Admin | Guest       |
+      | User        | Admin       |
+      | User        | Space Admin |
+      | User        | User        |
+      | User        | Guest       |
+      | Guest       | Admin       |
+      | Guest       | Space Admin |
+      | Guest       | User        |
+      | Guest       | Guest       |
+
+
+  Scenario Outline: user with different user role gets his/her own drive information
+    Given the administrator has assigned the role "<userRole>" to user "Alice" using the Graph API
+    When user "Alice" gets the personal drive information using Graph API
```

Comment:
```suggestion
    When user "Alice" gets own personal drive information using Graph API
```
if we could do something like this? since alice is getting his own information right?